src/modules/manage-team/sections/kudometer/KudometerRow.spec.tsx

```diff
@@ -5,15 +5,20 @@ import {
   findByTestId, mockLocalstorage, wait, withMockedProviders,
 } from '../../../../spec_helper';
 import { KudometerRow } from './KudometerRow';
-import { DELETE_KUDOMETER, GET_KUDOMETERS, Kudometer } from './KudometerQuerries';
+import {
+  DELETE_KUDOMETER, GET_KUDOMETERS, Kudometer, SET_ACTIVE_KUDOS_METER,
+} from './KudometerQuerries';
```

Comment:
`KudometerQueries` ?

Reply:
It's a separate file that contains all the GraphQL queries and interfaces because they have tendency to be quite long and clutter up your files.

Reply:
I meant to say, that there is a typo. Plural of `Query` is `Queries`, so `KudometerQuerRies` => `KudometerQueries` (single `r`)

Reply:
Totally missed that. I'm going to play the 'The file was already there' card on this one. I'll open an issue for this since this PR has already been closed.